src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -283,9 +292,17 @@ namespace plugins
     public: void SetSceneName(const std::string &_name);
 
     /// \brief Set the initial pose the render window camera
-    /// \param[in] _pose Initical camera pose
+    /// \param[in] _pose Initial camera pose
     public: void SetCameraPose(const math::Pose3d &_pose);
 
+    /// \brief Set the render window camera's near clipping plane distance
+    /// \param[in] _near Near clipping plane distance
+    public: void SetCameraNearClip(const double &_near);
+
+    /// \brief Set the render window camera's far clipping plane distance
+    /// \param[in] _far Far clipping plane distance
+    public: void SetCameraFarClip(const double &_far);
```

Comment:
for primitive types, I think we're now just doing pass by value instead of const ref, e.g. `double _far`

Reply:
Thanks! 06b7c185518d329f15c707cd922560f10bb4613f